website/docs/tutorials/deploying-java-app-with-database.md

```diff
@@ -0,0 +1,259 @@
+    ```
+
+10. Use the URL to navigate to the `CreatePerson.xhtml` data entry page to use the application:
+    In case of this tutorial, we will access `http://mysboproj-9080.192.168.49.2.nip.io/CreatePerson.xhtml`. Note that the URL could be different for you. Now, enter a name and age data using the form.
+11. Click on the **Save** button when complete
+
+
+Note that the entry of any data does not result in the data being displayed when you click on the "View Persons Record List" link.
+
+### Creating a database to be used by the sample application
+
+You can use the default configuration of the PostgreSQL Operator to start a Postgre database from it. But, since our app uses few specific configuration values, lets make sure they are properly populated in the database service we start.
+
+1. Store the YAML of the service in a file:
+    ```shell
+    odo service create postgresql-operator.v0.1.1/Database --dry-run > db.yaml
+    ```
+
+2. Modify and add following values under `metadata:` section in the `db.yaml` file:
+    ```yaml
+    name: sampledatabase
+    annotations:
+      service.binding/db_name: 'path={.spec.databaseName}'
+      service.binding/db_password: 'path={.spec.databasePassword}'
+      service.binding/db_user: 'path={.spec.databaseUser}'
+    ```
+
+    This configuration ensures that when a database service is started using this file, appropriate annotations are added to it. Annotations help the Service Binding Operator in injecting those values into the application. Hence, the above configuration will help Service Binding Operator inject the values for `databaseName`, `databasePassword` and `databaseUser` into the application.
+
+3. Change the following values under `spec:` section of the YAML file:
+    ```yaml
+    databaseName: "sampledb"
+    databasePassword: "samplepwd"
+    databaseUser: "sampleuser"
+    ```
+
+4. Create the database from the YAML file:
+    ```shell
+    odo service create --from-file db.yaml
+   ```
+   ```shell
+    odo push --show-log
+    ```
+
+    This action will create a database instance pod in the `my-postgresql-operator-dev4devs-com` namespace. The application will be configured to use this database.
+
+### Binding the database and the application
+
+Now, the only thing that remains is to connect the DB and the application. We will use odo to create a link to the Dev4Devs PostgreSQL Database Operator in order to access the database connection information.
+
+1. Display the services available to odo: - You will see an entry for the PostgreSQL Database Operator displayed:
+    ```shell
+   odo catalog list services
+   ```
+   Your output should look similar to the following:
+   ```shell
+   $ odo catalog list services
+   Operators available in the cluster
+   NAME                                             CRDs
+   postgresql-operator.v0.1.1                       Backup, Database
+    ```
```

Comment:
Not sure why we need `odo catalog` set of commands here. We should use `odo service list` to list the Database service we created in above sectoin.

Reply:
Since that's what point 2 in this section is about, IMO, we could skip the first part of `odo catalog list services` entirely.